xpk.py

```diff
@@ -1887,21 +1887,20 @@ def get_main_container(args, system, docker_image, command) -> str:
                    docker_image=docker_image,
                    command=command)
 
-def get_gke_outlier_dashboard(args):
-  """Get the identifier of GKE outlier dashboard deployed in the project.
+def get_gke_dashboard(args, dashboard_filter):
+  """Get the identifier of GKE dashboard deployed in the project.
 
   Args:
     args: user provided arguments for running the command.
 
   Returns:
     str:
-      identifier of outlier dashbord if deployed in project,
+      identifier of dashbord if deployed in project,
       None otherwise.
   """
-  outlier_dashboard_filter = "displayName:'GKE - TPU Monitoring Dashboard'"
   command = (
       'gcloud monitoring dashboards list'
-      f' --project={args.project} --filter="{outlier_dashboard_filter}" --format="value(name)" --verbosity=error'
+      f' --project={args.project} --filter="{dashboard_filter}" --format="value(name)" --verbosity=error'
   )
 
   return_code, return_value = run_command_for_value(command, 'GKE Dashboard List', args)
```

Comment:
Sorry for an aside, could you also add to the below error message:

```
    xpk_print(f'GKE Dashboard List request returned ERROR {return_code}')

```

to be

```
    xpk_print(f'GKE Dashboard List request returned ERROR {return_code}. If you see a permissions error, please see xpk readme for possible solutions. ')

```

Reply:
Yeah, that's a great idea.